src/utils/stark.ts

```diff
@@ -280,15 +289,15 @@ export function toTransactionVersion(
 /**
  * Convert Transaction version to Fee version or throw an error
  * @param {BigNumberish} [providedVersion] 0..3 number representing the transaction version
- * @returns {ETransactionVersion} the fee estimation version corresponding to the transaction version provided
+ * @returns {TransactionVersion} the fee estimation version corresponding to the transaction version provided
```

Comment:
`{TransactionVersion}` doesn't match the `ETransactionVersion | undefined` return type from the function signature (although I'm pretty sure it doesn't matter and the `@returns` types are mostly unused).